data/parse_smogon_stats.py

```diff
@@ -41,64 +41,25 @@ def get_smogon_stats_file_name(game_mode, month_delta=1):
 
 
 def get_pokemon_information(smogon_stats_url):
-    """Parses a Smogon stats document, such as: 'https://www.smogon.com/stats/2019-02/moveset/gen7ou-1825.txt'
-       Returns a dictionary containing the most likely spreads, items, and moves for each pokemon in order of likelihood
-    """
+    spreads = []
+    items = []
+    moves = []
+    abilities = []
```

Comment:
hmm it looks like you're defining these lists outside the main loop that goes through each Pokemon. These lists should be refreshed every time a new pokemon is looked at otherwise there will only be one list containing _every_ pokemon's attributes :scream: